pkg/cmd/server/api/types.go

```diff
@@ -83,6 +95,8 @@ const (
 	ControllersAll = "*"
 )
 
+type FeatureSet map[OpenShiftFeature]util.Empty
```

Comment:
> Does the order really matter for feature disablement?

Having a config file that will end up with unstable reordering of values when round-tripped through a server is annoying.  It means that round-tripping a config to pick up new default values will not be comparable via diff.